src/integrators/wrapper_integrator.cpp

```diff
@@ -12,19 +12,17 @@ LinearToNonlinearFormIntegrator::LinearToNonlinearFormIntegrator(std::shared_ptr
 {
 }
 
-void LinearToNonlinearFormIntegrator::AssembleElementVector(const mfem::FiniteElement &                 el,
-                                                            mfem::ElementTransformation &               Tr,
-                                                            __attribute__((unused)) const mfem::Vector &elfun,
-                                                            mfem::Vector &                              elvect)
+void LinearToNonlinearFormIntegrator::AssembleElementVector(const mfem::FiniteElement &  el,
+                                                            mfem::ElementTransformation &Tr, const mfem::Vector &,
+                                                            mfem::Vector &               elvect)
 {
   m_f->AssembleRHSElementVect(el, Tr, elvect);
   elvect *= -1.;
 }
 
-void LinearToNonlinearFormIntegrator::AssembleElementGrad(const mfem::FiniteElement &                 el,
-                                                          mfem::ElementTransformation &               Tr,
-                                                          __attribute__((unused)) const mfem::Vector &elfun,
-                                                          mfem::DenseMatrix &                         elmat)
+void LinearToNonlinearFormIntegrator::AssembleElementGrad(const mfem::FiniteElement &  el,
+                                                          mfem::ElementTransformation &Tr, const mfem::Vector &,
+                                                          mfem::DenseMatrix &          elmat)
```

Comment:
Was this a compiler warning or just a cleaner way to do it?

Reply:
Just a cleaner way to do it. Also, in this particular case the variable actually was used, so it's also more correct.